content/en/metrics/faq/update-to-query-value-widget.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Update to Distribution Metrics Workflow FAQ
+kind: faq
+is_beta: true
+---
+
+## What is changing?
+
+We have unified the way metric queries are evaluated for Top Lists and Query Value Widgets such that you will have more consistent results for the same metric viewed in a Top List and a Query Value Widget. 
+
+As a refresher, regardless of the type of widget, each metric query consists of the same two evaluation steps to start: 1. Time aggregation, 2. Space aggregation (refresher [here][1]). 
+
+* A Query Value Widget simply reduces the results of those two steps into a single value.
+* A Top List Widget returns a single value per group.
+
+So we’ve unified the evaluation steps used for the Top List Widget to match the Query Value Widget’s (as shown in the table below):
+
+
+  | Widget                               | Old Evaluation Steps                                                                                                     | New Evaluation Steps             |
+|-------------------------------------------|----------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
+| Query Value Widget                              | 1. Time Aggregation | No Change  |
+| Top List | Evaluation Steps Here | Now matches Query Value widget|
```

Comment:
I can't tell what "here" refers to in "Evaluation steps here"

Reply:
Hi there!
I had a bit of trouble with formatting the table but its contents should look like this https://a.cl.ly/4gulAOxA

Reply:
I've pushed an update to your table contents; it might need some adjusting after we see the staging site build.